tests/deepsparse/transformers/pipelines/test_text_generation.py

```diff
@@ -12,285 +12,236 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import List, Optional, Tuple
+import os
+from typing import List, Optional, Tuple, Union
```

Comment:
- model by model (each config has their own file) (stub, model_name, use_bos_token, max_diff).
- cadence
- scenarios: turn off and off the essential files.

Pytest.any to define configs. pytest.ini